index.html

```diff
@@ -15,15 +15,148 @@
 
     <!-- We are using a CSS Reset file to standardize browser default css -->
     <link rel="stylesheet" href="reset.css" />
-
-    <!-- This is where all of our awesome CSS code goes -->
     <link rel="stylesheet" href="style.css" />
+    <link href="Mina/Mina-Regular.ttf" rel="stylesheet"> <!-- this is the logo-->
   </head>
```

Comment:
There is a reference to a `Mina` folder, but no `Mina` folder exists in your GitHub repo, so you are getting a `404` broken link error here.